torch/testing/_internal/common_device_type.py

```diff
@@ -20,6 +20,10 @@
 except ImportError:
     HAS_PSUTIL = False
 
+IS_SANDCASTLE = os.getenv('SANDCASTLE') == '1' or os.getenv('TW_JOB_USER') == 'sandcastle'
```

Comment:
I think you can import at least two of these, and the third one should probably be added to and imported from the same place. See: 

https://github.com/pytorch/pytorch/blob/789e9353044eb2227462ef470e4b362ac025be4d/torch/testing/_internal/common_utils.py#L61

Reply:
sounds good. will move those to `common_utils`